styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerTest.java

```diff
@@ -117,10 +123,11 @@ public void testGetKubernetesClient() throws Exception {
         "styx.gke.foo.cluster-id", cluster,
         "styx.gke.foo.namespace", namespace));
 
+
     final String endpoint = "k8s.example.com:4711";
-    final String clusterCaCertificate = "cluster-ca-cert";
-    final String clientCertificate = "client-cert";
-    final String clientKey = "client-key";
+    final String clusterCaCertificate = Resources.toString(Resources.getResource("ca.crt"), UTF_8);
```

Comment:
So creating Http client manually requires proper CAs and key? What is the implication of production?